portal/views/portal.py

```diff
@@ -825,14 +834,17 @@ def config_settings(config_key):
     )
     if config_key:
         key = config_key.upper()
-        if not any(key.startswith(prefix) for prefix in config_prefix_whitelist):
+        if not any(
+            key.startswith(prefix) for prefix in config_prefix_whitelist
+        ):
             abort(400, "Configuration key '{}' not available".format(key))
         return jsonify({key: current_app.config.get(key)})
     else:
```

Comment:
Can you remove the `else` and un-indent this section? the `if` block is guaranteed to return/exit so `else` isn't necessary